### DISCUSSION TITLE
Welcome to Custom Tkinter Builder Discussions

### DISCUSSION THREAD

**Discussion by rigvedmaanas** [rigvedmaanas/CustomTkinterBuilder#1] [Announcements]
Here you can discuss questions, suggest ideas, and showcase your projects made with Custom Tkinter Builder.
For reporting bugs please use the issues tab.

**Comment by sbulazel** [comment]
Hi! I just installed the Builder, started it and 1) tried to create a new project and 2) tried to open an example. But nothing works. I'm working with a Mac (OS 15.4.1). The only thing i can see, is the the very first dialog (create project, open project) ...

**Comment by paweljan92** [comment]
Hi! Unfortunatelly I'm not able to open any file. I think it has something to do with the path, which is shown as the pattern in title: C:/../../..\ProjectFolder
So the last sign is a backslash not a slash
I'm using windows 10.
